connect/src/protocols/cctp/cctpTransfer.ts

```diff
@@ -584,10 +585,14 @@ export namespace CircleTransfer {
     const dstToken = Wormhole.chainAddress(dstChain.chain, dstUsdcAddress);
     const srcToken = Wormhole.chainAddress(srcChain.chain, srcUsdcAddress);
 
+    // https://developers.circle.com/stablecoins/docs/required-block-confirmations
+    const eta =
+      srcChain.chain === "Polygon" ? 2_000 * 200 : finality.estimateFinalityTime(srcChain.chain);
```

Comment:
@kev1n-peters Is this going to be a fixed ETA when it's Polygon?

Reply:
I mean it says 200 blocks in the linked document, but is it 2 secs per block as fixed time in Polygon?

Reply:
Yeah, it's always going to be a fixed time (`estimateFinalityTime` also returns a "fixed time", but we are overriding the finality time for Polygon here since it's different for CCTP transfers than for token bridge transfers